java/src/jmri/jmrix/loconet/LocoNetBundle.properties

```diff
@@ -135,3 +137,6 @@ StatusWriting = write SV{0} (port {1} V{2})
 StatusWriteMode = write SV{0} (port {1} mode)
 AddressToolTip = Enter LocoIO Module Low Address (hex 01-7f, excluding 0x50 = dec 80)
 SubAddressToolTip = Enter LocoIO Module High Address (hex 01-7f = dec 1-126)
+
+# Used by several different LocoNet-based tools
+ButtonCancel = Stop
```

Comment:
If a button with the text `Stop` is desired, I suggest that the bundle is `ButtonStop`. Buttons with the word `Cancel` is heavily used in software, so I think it's bad to use `ButtonCancel` for anything else than `Cancel`.

Reply:
Changed it to get the "inherited" `ButtonCancel` value, which somehow works just fine right now (it failed last nite!)